profile_label/profile_label/handlers.py

```diff
@@ -0,0 +1,38 @@
+import os
+import json
+
+# from jupyter_server.base.handlers import APIHandler
+from notebook.base.handlers import APIHandler
+# from jupyter_server.utils import url_path_join
+from notebook.utils import url_path_join
+
+import tornado
+from tornado.web import StaticFileHandler
+
+class RouteHandler(APIHandler):
+    # The following decorator should be present on all verb methods (head, get, post,
+    # patch, put, delete, options) to ensure only authorized user can request the
+    # Jupyter server
+    @tornado.web.authenticated
+    def get(self):
+        profile_name = os.environ.get('opensarlab_profile_name', 'No environment variable "opensarlab_profile_name" found!')
```

Comment:
Should this be `opensarlab_profile_name` or `osl_profile_name` or `OPENSARLAB_PROFILE_NAME` or whatever? We need to be consistent so other extensions can use the same env.

Reply:
updated